source/includes/_attempts.md

```diff
@@ -381,6 +663,143 @@ Parameter | Description
+  "user": {
+    "id": 5,
+    "url": "http://demo.testpress.in/api/v2.2/users/5/",
+    "username": "demouser",
+    "display_name": "DemoUser is my name eeeeee123456789012345678901234",
+    "first_name": "DemoUser is my name eeeeee123456789012345678901234",
+    "last_name": "",
+    "photo": "https://media.testpress.in/i/bc194c3b0f3241ad9473edf949de3cec.jpeg",
+    "large_image": "https://s3-ap-southeast-1.amazonaws.com/media.testpress.in/i/459b2af2355549dc8d7f411553128903.jpeg",
+    "medium_image": "https://s3-ap-southeast-1.amazonaws.com/media.testpress.in/i/05c3efc7e4e3454ebad1c32c0c69297c.jpeg",
+    "medium_small_image": "https://s3-ap-southeast-1.amazonaws.com/media.testpress.in/i/ca7ae15e26cc4586ae5f98750c2acc62.jpeg",
+    "small_image": "https://s3-ap-southeast-1.amazonaws.com/media.testpress.in/i/33936fd431e14a9ab54d7e24e70bf301.jpeg",
+    "x_small_image": "https://s3-ap-southeast-1.amazonaws.com/media.testpress.in/i/673ceb6e18b343318fba39279ef07594.jpeg",
+    "mini_image": "https://s3-ap-southeast-1.amazonaws.com/media.testpress.in/i/3214f5359ad247d1b2e38609376af72b.jpeg"
+  },
+  "date": "2017-05-24T07:32:09.891Z",
+  "total_questions": 50,
+  "score": "0.00",
+  "review_pdf_url": "",
+  "review_url": "http://demo.testpress.in/api/v2.2/attempts/400/review/",
+  "questions_url": "http://demo.testpress.in/api/v2.2/attempts/400/questions/",
+  "percentile": 0,
+  "correct_count": 0,
+  "incorrect_count": 0,
+  "last_started_time": "2017-05-24T07:37:10.655Z",
+  "remaining_time": "1:54:59",
+  "time_taken": "0:05:01",
+  "state": "Running",
+  "rank": "NA",
+  "max_rank": "NA",
+  "percentage": "0",
+  "unanswered_count": 0,
+  "commented_questions_count": 0,
+  "comments_count": 0,
+  "total_bonus": 0,
+  "rank_enabled": false,
+  "exam": {
+    "url": "http://demo.testpress.in/api/v2.2/exams/dummy-test-7-8/",
+    "id": 31,
+    "title": "Dummy Test 7",
+    "description": "",
+    "start_date": "2017-03-30T18:58:53+07:00",
+    "end_date": null,
+    "duration": "2:00:00",
+    "number_of_questions": 50,
+    "negative_marks": "0.00",
+    "mark_per_question": "1.00",
+    "template_type": 1,
+    "allow_retake": true,
+    "max_retakes": -1,
+    "enable_ranks": false,
+    "rank_publishing_date": null,
+    "allow_pdf": false,
+    "allow_question_pdf": false,
+    "created": "2017-03-30T13:29:05.007Z",
+    "slug": "dummy-test-7-8",
+    "variable_mark_per_question": false,
+    "show_answers": true,
+    "allow_preemptive_section_ending": false,
+    "sections": [
+      {
+        "order": 0,
+        "name": "",
+        "duration": "2:00:00",
+        "cut_off": 0
+      }
+    ],
+    "immediate_feedback": false,
+    "categories": []
+  },
+  "sections": [],
+  "speed": 0,
+  "accuracy": 0,
+  "institute_attempt_id": null
+}
+
+```
+
+For every 1 minute this API have to be called when the user attempting an exam to update the remaining time in server.
```

Comment:
For every minute, this API should be called during the attempt window. This is done to update the remaining time for that attempt in the server.